code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -106,6 +106,22 @@
 	damage_mult = CONFIG_GET(number/combat_define/base_hit_damage_mult) + CONFIG_GET(number/combat_define/low_hit_damage_mult)
 	recoil_unwielded = CONFIG_GET(number/combat_define/min_recoil_value)
 
+/obj/item/weapon/gun/pistol/m1911/custom
+	name = "\improper M1911A1 Custom pistol"
+	desc = "A 20th century military firearm ancient pistol that received several modifications. The feeding ramp is polished to a mirror sheen. The slide's been reinforced and the interlock with the frame is tightened for added precision. The sight system is original and the thumb safety is extended to make it easier on the finger. A long-type trigger with non-slip grooves and a ring hammer has been installed. The base of the trigger guard's been filed down for a higher grip and not only that, nearly every part of this gun has been expertly crafted and customized"
```

Comment:
I'm totally not against the reference, but it really ought to be moved to the codex whenever possible. 
Rule of thumb: It shouldn't be longer than the average costum emote on a HRP server.

Reply:
lol